tests/unit/v1/test_bundle.py

```diff
@@ -441,7 +442,10 @@ def test_deserialize_from_seconds_nanos(self):
         )
 
         self.assertRaises(
-            ValueError, _helpers.deserialize_bundle, _serialized, client=client,
+            (ValueError, ParseError),  # protobuf 3.18.0 raises ParseError
```

Comment:
Should deserialize be changed to provide a better error here? Otherwise our method is now throwing a parse error where it previously didn't? Or perhaps at least we could include additional info?

Luckily bundles is a recent add and likely not many folks are using it. Less folks even that are going between node + python serialization (what this is about)

Reply:
Chatted about with @tseaver. This is the error we get now, as opposed to our custom value error. As it tells the user what is going on, we don't try to wrap any longer.
```
google.protobuf.json_format.ParseError: Failed to parse createTime field: Timestamp JSON value not a string: {'seconds': '1616434660', 'nanos': 913764000}.
```

https://github.com/googleapis/python-firestore/blob/51c92aa45156c5462f35991f65c02dcac49aacc3/google/cloud/firestore_v1/_helpers.py#L1202-L1211

Reply:
I'd like to remove this odd code that is no longer relevant, but without us depending direct on protobuf I don't think we can do that.

Reply:
Can we add a comment though that is just to handle protobuf 3.17?

Reply:
I think we can make the case that the existing "rewrap AttributeError" code this test is exercising is there for "improved DX", rather than because we are making a Javiotic promise that we will only allow ValueError to propagate from `deserialize_bundle`.  Because the new `ParseError` tells the user what is wrong with their input:

```Python
E       ValueError: Timestamp JSON value not a string: {'seconds': '1616434660', 'nanos': 913764000}
```
I think we can declare that as a "DX win," and go on.